### PR TITLE
ZEPPELIN-490 Tidying up dynamicform.md to distinguish each category e…

### PR DESCRIPTION
**Pull request by AhyoungRyu**
This PR is related to [this issue](https://issues.apache.org/jira/browse/ZEPPELIN-490?jql=project%20%3D%20ZEPPELIN).
Current `dynamicform.md` is little bit hard to distinguish each category.
So, I applied `nav-tabs` by using Bootstrap. 

 - Before applying this PR
before

 - After 
after1
after2

### CODE REVIEW

**Comment by felixcheung** [comment]
nice. if I choose python on the top tab do other tab switch to python too?
also, seems like syntax highlighting is off?

**Comment by AhyoungRyu** [comment]
Thank you for review @felixcheung ! ☺

Because of *same language code tab showing* and *code highlighter*, as you mentioned, I made a new  javascript file : `docs.js`.

So, this time result is
fillpr

**Comment by Leemoonsoo** [comment]
Working well. 
Looks good to me

**Comment by Leemoonsoo** [comment]
Merge if there're no more discussions

File: docs/manual/dynamicform.md [hunk 1]
```diff
@@ -58,75 +54,83 @@ Also you can separate option's display name and value, using _${formName=default
...
+<div class="codetabs">
+    <div data-lang="scala" markdown="1">
 
-Scala
-
-```scala
+{% highlight scala %}
 %spark
 println("Hello "+z.select("day", Seq(("1","mon"),
+                                        ("2","tue"),
```
Comment (felixcheung) [inline docs/manual/dynamicform.md]: I think there's a slight indentation offset here and before, as you can see in the final html output.
Could you check?

Comment (AhyoungRyu) [inline docs/manual/dynamicform.md]: Oh that was my bad. 
I fixed it : )

**Comment by felixcheung** [comment]
great! thanks for making this so much better!

**Comment by Leemoonsoo** [comment]
Merge if there're no more discussions

**Pull request closed without merging**